frontend/institution/institution.js

```diff
@@ -59,10 +59,16 @@ Institution.prototype.addChildInst = function addChildInst(institution){
     childIsNew ? this.children_institutions.push(institution) : null;
 };
 
+Institution.prototype.getSimpleAddress = function () {
+    if(this.address) {
```

Comment:
What if there is no address? Remove the if statement or add other return to the else case